ci: failure in wallet_taproot.py

https://cirrus-ci.com/task/4517217436958720.
```bash
 node0 2022-03-07T17:49:17.320927Z [httpworker.0] [rpc/request.cpp:179] [parse] ThreadRPCServer method=getnewaddress user=__cookie__ 
 test  2022-03-07T17:49:17.353000Z TestFramework (ERROR): Assertion failed 
                                   Traceback (most recent call last):
                                     File "/tmp/cirrus-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/test_framework.py", line 132, in main
                                       self.run_test()
                                     File "/tmp/cirrus-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/wallet_taproot.py", line 435, in run_test
                                       lambda k1, k2: (key(H_POINT), [multi_a(1, [k1, k2])])
                                     File "/tmp/cirrus-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/wallet_taproot.py", line 338, in do_test
                                       self.do_test_psbt(comment, pattern, privmap, treefn, keys[2*nkeys:3*nkeys], keys[3*nkeys:4*nkeys])
                                     File "/tmp/cirrus-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/wallet_taproot.py", line 317, in do_test_psbt
                                       assert_equal(addr_g, addr_r)
                                     File "/tmp/cirrus-build/ci/scratch/build/bitcoin-x86_64-pc-linux-gnu/test/functional/test_framework/util.py", line 51, in assert_equal
                                       raise AssertionError("not(%s)" % " == ".join(str(arg) for arg in (thing1, thing2) + args))
                                   AssertionError: not(bcrt1pelz2cyn9xkvu80huenyf9qenm55964tl3m0dqjshaj5jcagas7zsdezgpl == bcrt1pu8csjm3g7sqdwumtk86etzcrtsq543kdxvtd8vu7mtjdcsnckvvs3p693c)
```

Fixed by #24490.

--- a/test/functional/wallet_taproot.py
+++ b/test/functional/wallet_taproot.py
@@ -429,10 +429,10 @@ class WalletTaprootTest(BitcoinTestFramework):
             lambda k1, k2: (key(H_POINT), [multi_a(1, [k1, k2], True)])
         )
         self.do_test(
-            "tr(H,multi_a(1,XPUB,XPRV))",
-            "tr($H,multi_a(1,$1/*,$2/*))",
+            "tr(H,{H,multi_a(1,XPUB,XPRV)})",
+            "tr($H,{pk($H),multi_a(1,$1/*,$2/*)})",
             [False, True],
-            lambda k1, k2: (key(H_POINT), [multi_a(1, [k1, k2])])
+            lambda k1, k2: (key(H_POINT), [pk(H_POINT), [multi_a(1, [k1, k2])]])
         )
         self.do_test(
             "tr(H,sortedmulti_a(1,XPUB,XPRV,XPRV))",

